website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -0,0 +1,75 @@
+---
+layout: docs
+page_title: Ingress Controller Integrations
+description: Configuring Ingress Controllers With Consul On Kubernetes
+---
+
+# Configuring Ingress Controllers with Consul on Kubernetes
+
+-> This topic requires familiarity with [Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+
+-> The following information requires Consul 1.10+, Consul-k8s 0.26+, Consul-helm 0.32+ configured with
+[Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
+
+This page describes a general approach for integrating Ingress Controllers with Consul on Kubernetes to secure traffic from the Controller
+to the backend services. This allows Consul to transparently secure traffic from the ingress point through the entire traffic flow of the service.
+
+A few steps are generally required to enable an Ingress controller to join the mesh and pass traffic through to a service:
+
+* Enable connect-injection via an annotation on the Ingress Controller's deployment: `consul.hashicorp.com/connect-inject` is `true`.
+
+* Using the following annotations on the Ingress controller's deployment, setup exclusion rules for it's ports.
+  * [`consul.hashicorp.com/transparent-proxy-exclude-inbound-ports`](https://www.consul.io/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-inbound-ports) - Provides the ability to exclude a list of ports for
```

Comment:
It might help to call out that if your Ingress Controller uses a `Type: LoadBalancer` service, you must add container port exclusions to this list to allow the load balancer to communicate with the services.

Reply:
I think an example of a real spec here would be immensely useful.